packages/ng-table-builder/src/table-builder.component.ts

```diff
@@ -307,6 +308,20 @@ export class TableBuilderComponent<T>
         }
     }
 
+    public updateColumnsSchema(patch: Nullable<PlainObjectOf<Partial<ColumnsSchema>>>): void {
+        if (isNotNil(patch)) {
```

Comment:
copy paste?

Reply:
![image](https://user-images.githubusercontent.com/12021443/124121338-73976180-da7d-11eb-9f0a-1fa65a948b07.png)

Reply:
i have simplified my function because there is no difference in behavior